rohmu/notifier/http.py

```diff
@@ -130,7 +130,7 @@ def close(self) -> None:
         self._stop_event.set()
         self._thread.join(_THREAD_JOIN_TIMEOUT)
 
-    def object_created(self, key: str, size: Optional[int]) -> None:
+    def object_created(self, key: str, size: Optional[int], metadata: Optional[dict]) -> None:
```

Comment:
Perhaps this signature could just have mandatory key and optional kwargs that default to None? With that, the callers would look bit cleaner.

Reply:
yes, the signature is making the calls smelly. Ideally, these would be dedicated types (`NotifyObjectCreated`, ....) (probably eventually when the API requirements stabilizes). 
kwargs for me is a classic way to forget passing additional arguments even if they are available.